buildsrc/compile.xml

```diff
@@ -63,7 +63,11 @@
     <target name="-process-test-resources" if="test.resources.dir.exsits">
         <macro:processresources dir="${src.test.resources.dir}" todir="${build.test.dir}"/>
     </target>
+	
+	<target name="-process-meta-inf-resources" if="meta_inf.dir.exists">
+	    <macro:processresources dir="${meta_inf.dir}" todir="${build.main.dir}/META-INF"/>
+	</target>
```

Comment:
I don't think this, or any of the ant changes are needed if the manifest file is just kept where it is.